src/cascadia/TerminalApp/SuggestionsControl.cpp

```diff
@@ -508,7 +508,7 @@ namespace winrt::TerminalApp::implementation
                         automationPeer.RaiseNotificationEvent(
                             Automation::Peers::AutomationNotificationKind::ItemAdded,
                             Automation::Peers::AutomationNotificationProcessing::MostRecent,
-                            paletteItem.Name() + L" " + paletteItem.KeyChordText(),
+                            paletteItem.Name(),
```

Comment:
This is a change in behavior, but I'm ok with it assuming we don't display the key chord in the suggestions UI

Reply:
Yep we don't